index.html

```diff
@@ -1187,32 +1188,37 @@ <h3>Identifiers</h3>
 When expressing statements about a specific thing, such as a person, product,
 or organization, it is often useful to use some kind of identifier so that
 others can express statements about the same thing. This specification defines
-the optional <code>id</code> <a>property</a> for such identifiers. The <code>id</code>
-<a>property</a> is intended to unambiguously refer to an object such as a person,
-product, or organization. This allows for the expression of statements about
-specific things in the verifiable credential. <em>If</em> the <code>id</code>
-<a>property</a> is present:
-        </p>
-	<ul>
-	  <li>
-the <code>id</code> <a>property</a> MUST express an identifier that
-others are expected to use when expressing statements about specific
-thing identified by that identifier
-	  </li>
-	  <li>
-the <code>id</code> <a>property</a> MUST NOT have more than one value
-	  </li>
-	  <li>
-the value of the <code>id</code> <a>property</a> MUST be a URI
-	  </li>
-	</ul>
+the optional <code>id</code> <a>property</a> for such identifiers. The
+<code>id</code> <a>property</a> is intended to unambiguously refer to an object,
+such as a person, product, or organization. Using the <code>id</code>
+<a>property</a> allows for the expression of statements about specific things in
+the <a>verifiable credential</a>.
+        </p>
+
+        <p>
+<em>If</em> the <code>id</code> <a>property</a> is present, the:
```

Comment:
"the" belongs at the start of each bullet.  Comprehension is not increased by this split.

Reply:
Repeating text at the beginning of each bullet point, when you can have it just once in the stem sentence goes against my pedantic nature of being a Technical Writer, but done. :smile: